README.md

```diff
@@ -1 +1,113 @@
-# transact_track
\ No newline at end of file
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#About-the-Project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [Setup](#setup)
+- [✈  Live](#Live-Demo)
+- [😎 Loom](#Loom-video)
+- [👥 Authors](#Authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#Show-your-support)
+- [🙏 Acknowledgements](#Acknowledgments)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# [About the Project] <a name="TransactionTrack"></a>
+
+**[TransactionTrack]**TransactionTrack is a powerful app designed to help you effortlessly record and track all your transactions for a specific item. Whether you're managing your personal finances, monitoring business expenses, or simply keeping tabs on your spending habits, this app is the perfect companion to stay organized and informed.
+
+##  Built With <a name="RoR"></a>
+### Tech Stack <a name="tech-Stack"></a>
+- **[RoR]**
+- **[Html&Css]**
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/RoR">RUBY</a></li>
+  </ul>
+</details>
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/html&css">RAILS</a></li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[HOME]**
+- **[CATEGORIES]**
+- **[TRANSACTIONS]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Setup
+
+Clone this repository to your desired folder using the following command; git clone https://github.com/ismailmunyentwari9/transact_track.git
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LIVE DEMO -->
+
+## LIVE DEMO <a name="Live-Demo"></a>
+- For live Demo of my project please  click <a href="https://transaction-truck.onrender.com">Live</a>
+<!-- LOOM VIDEO -->
+
+## LOOM VIDEO <a name="Loom-Video"></a>
+- For Loom Video of my project please  click <a href="https://www.loom.com/share/dc35944203bd4bf88a5e90eba408ce30?sid=682e3faa-c221-4c6e-a1d2-44d53de70461">Live</a>
```

Comment:
- [ ] Kindly consider re-recording the video for your project and make sure to highlight a piece of the code the student finds interesting or is proud of making as stated in the project requirements
>![image](https://github.com/ismailmunyentwari9/transact_track/assets/53402600/36549e91-4c02-43a8-80a5-32a1d519d4dc)